src/platform_interface.rs

```diff
@@ -11,7 +11,13 @@ pub(crate) trait PlatformInterface {
   ) -> Result<Command, OutputError>;
   
   /// Get generate file ext
-  fn get_recipe_file_ext(command: &str) -> String;
+  fn get_script_file_ext(command: &str) -> &str {
```

Comment:
Let's call this `get_script_file_suffix`, to match `std::env::consts::EXE_SUFFIX`.
```suggestion
  fn get_script_file_suffix(command: &str) -> &str {
```